plugins-bundle-analyzer/compareBundles.js

```diff
@@ -12,20 +12,22 @@ let savingBundles = {},
   grewDownMessage = '';
 
 const generatePRComment = () => {
-  let message = newBundles
-    ? 'New packages found.\nPlease update the baseline file by running locally "npm run analyzeBundles" and push the changes.\n\n'
-    : '';
-  message += grewDownMessage ? `Packages that shrank:\n${grewDownMessage}\n` : '';
+  let message = 'Comparison bundleSizes:\n';
+  if (!grewUpMessage && !grewDownMessage && !newBundles) {
+    message += 'Comparison ended successfully.\n';
```

Comment:
No changes in Bundles sizes :)
```suggestion
    message += 'No changes in Bundles sizes.\n';
```